example_computer_setup/computers/my-cluster.yaml

```diff
@@ -12,4 +12,5 @@ prepend_text: " "
 append_text: " "
 # Extra properties
 extras:
-    nfs_hostname: "nfs.mycluster.whatever"
+   # note: will be autodetected if not specified
+   conda_dir: "/home/myuser/miniconda3"
```

Comment:
Perhaps this would be a good time to support `{username}` here

Reply:
Ah right, it's actually already supported ;-)

Reply:
Does that mean we can indeed change this?